### PR TITLE
Alert levels affect your mood now! :)

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Delta gives you a speed boost and negative moodlet if delta is cancelled you get a lesser negative moodlet and slowdown

## Why It's Good For The Game
Helps gives a boost to run to a pod, goes well with the banging techno muisc

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image
image

(run in this order, so the second red alert was downgrading from delta)

</details>

## Changelog
:cl:
add: Security levels affect your mood!
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
I don't want my mood management go awful...

**Comment by etherware-novice** [comment]
wdym

**Comment by EvilDragonfiend** [comment]
Having global -2 mood is very awful nerf to non-antags. Non-antags should have more mood managemenr while they have red alert issue.

**Comment by etherware-novice** [comment]
it checks if youre a syndicate i think

**Comment by Coding-Grape** [comment]
That wasn't the point
Having a global -2 mood in a red alert situation is fucking terrible from a gameplay perspective, as it's *red alert* and you're doing shit and can't afford to get the mood penalties from low mood OR constantly micromanaging your mood with mood buffs while trying to make it NOT a red alert situation

**Comment by etherware-novice** [comment]
-1?

**Comment by Rukofamicom** [comment]
Brilliant PR that adds a strong incentive not to stay in unnecessarily elevated alert levels.

I think the mood levels are fine. Red alert **should** be stressful for general crew and the mood penalties are a great way to convey that into the gameplay. I like that Blue alert doesn't have any bonuses in either direction.

I think -2 for red alert is perfectly fine.

**Comment by etherware-novice** [comment]
well i just completely replaced it so,,,

**Comment by EvilDragonfiend** [comment]
I think syndi should care as well unless they are murderboner. Use `is_murderbone()` proc.

**Comment by RDS88-beeman** [comment]
Shouldn't coming down from delta be positive, because "holy shit we're not going to die immediately"

**Comment by EvilDragonfiend** [comment]
> Shouldn't coming down from delta be positive, because "holy shit we're not going to die immediately"

It is intentional. Crews feel exhausted when they are out of delta movement spred buff.

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -47,6 +47,38 @@
...
+/datum/mood_event/delta
+	description = "<span class='boldwarning'>RUN!! IT'S GOING TO BLOW!!</span>"
+	mood_change = -6
+	timeout = 4 MINUTES  // most likely manually cleared but in case..
+
+/datum/mood_event/delta/add_effects()
+	var/datum/component/L = owner
+	var/mob/living/crew = L.parent
+
+	crew.add_movespeed_modifier("delta_panic", multiplicative_slowdown=-1.3)
```
Comment (PowerfulBacon) [inline code/datums/mood_events/generic_negative_events.dm:59]: Have you tested how fast this is? This seems massively quick

**Comment by PowerfulBacon** [comment]
Stale

**Pull request closed without merging**